pkg/collector/corechecks/cluster/orchestrator.go

```diff
@@ -0,0 +1,415 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
could you move this to a dedicated `orchestrator` sub-package?  see https://github.com/DataDog/datadog-agent/pull/7433